.ansible-lint

```diff
@@ -20,11 +18,8 @@ rulesdir:
   - ./.ansible-lint-rules/
 skip_list:
   - galaxy[no-changelog]
-  - ignore-errors
-  - no-handler
-  - package-latest
```

Comment:
I would prefer to keep this rule in the skip_list, as we have no intention of following it ever.

Reply:
The rule that you want to skip is our own rule, which we had include because we want to use all fqcn in all repos. I actually work on this issue.

Reply:
Done